securedrop/tests/test_store.py

```diff
@@ -32,30 +32,29 @@ def create_file_in_source_dir(config, filesystem_id, filename):
     return source_directory, file_path
```

Comment:
To further simplify the tests in this file, I would recommend:

- Removing all mentions of the journalist_app, as it is out of scope for what is being tested here (only the Storage class is being tested; not the full app).
- Not using the storage fixture as I would argue that creating a Storage class should be done in the test code as that's what's being tested: it's not "setup" code. The benefit will be that the tests won't be coupled with the config fixture (which is used in a lot of tests and seems unnecessary here) and the tests will be easier to read/follow.

You can see an example of tis approach above in TestPassphrasesGenerator.

Reply:
The `journalist_app` fixture is unused in the majority of the tests alright - there are a few that do require an app context - eg. the shredder tests, but they could probably be moved to a separate file.

I'm less convinced about removing the `app_storage` fixture in all cases. Some of the tests that use it and the `config` fixture are essentially verifying that files are being created in the store in the expected location based on config parameters, and for those it would seem either tautological to just remove `config`, or duplicative to effectively recreate both fixtures in the test. But it can definitely be removed from tests not using `config` and replaced with a test-specific Storage object.

Reply:
Actually on further reflection there are a lot more cases where the latter makes sense...